public/Get-DbaBackupInformation.ps1

```diff
@@ -273,6 +273,24 @@ function Get-DbaBackupInformation {
                             Write-Message -Level VeryVerbose -Message "File"
                             $Files += $f.FullName
                         }
+                    } elseif ($f -match "^http") {
```

Comment:
I can foresee a nice WTF if you call this method with a SAS URL for a container, and you have NOT created the SAS credential in MSSQL, because this method will properly enumerate the individual blobs, but when calling Read-DbaBackupHeader, they are going to get a permission denied issue (I could enumerate, but I could not read them...)